### DISCUSSION TITLE
Can't reload theia when workspace has the .tree file

### DISCUSSION THREAD

**Discussion by fengwang913** [eclipse-theia/theia#13801] [Q&A] [answered]
Hey,
1.
 I generated a treeEditor extension using the yo theia-extension command and was able to run the treeEditor extension successfully.
![success](https://github.com/eclipse-theia/theia/assets/65167225/3e5b2c4d-ceca-4913-9bed-44ebe88b1114)

 Then, I created a new file named tree-example.tree using the "New Example File" option under the Tree Editor menu. Everything was fine up to this point. However, when I refresh the webpage at this stage, the page fails to reopen. The error message is as follows:
![error](https://github.com/eclipse-theia/theia/assets/65167225/7ec2242a-8dd8-4a4f-8ec8-59e35dfc39d0)

2.
In the treeEditor extension, clicking on a .tree file doesn't render any elements by default. However, I would like it to render specific elements by default. For example, I want the user to click on a .tree file, have ControlUnit selected by default, and render the ControlUnitView. How can I achieve this?

Any help would be appreciated.

**Comment by msujew** [comment, marked as answer]
Hey @fengwang913,

we recently had a breaking change in the `Saveable` interface that leads to this error. The dependency for the widget is outdated. Please open an issue with the developers of that package.

**Reply by fengwang913** [reply]
Appreciate your suggestion!  I've opened the issue #13803 to track this problem.
